layers/+lang/plantuml/packages.el

```diff
@@ -0,0 +1,27 @@
+;;; packages.el --- plantuml layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Robert O'Connor <robby.oconnor@gmail.com>
+;; URL: https://github.com/robbyoconnor
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
+
+;;; Code:
+
+(defconst plantuml-packages
+  '(puml-mode))
+
+(defun plantuml/init-puml-mode ()
+  (use-package puml-mode
+    :defer t
+    :mode ("\\.puml" . puml-mode) ("\\.plantuml" . puml-mode)
```

Comment:
FWIW it seems we use parenthesis around the `:mode` arguments, like in https://github.com/syl20bnr/spacemacs/blob/develop/layers/+lang/c-c++/packages.el#L68